src/test/java/com/ibm/cloud/sdk/core/test/service/ResponseTest.java

```diff
@@ -166,74 +176,124 @@ public void testExecuteTestModel() throws InterruptedException {
   public void testEnqueue() throws InterruptedException {
     server.enqueue(new MockResponse().setBody(testResponseBody1));
 
+    final Map<String, Object> results = new HashMap<>();
```

Comment:
The only change to this test method is how the results are now saved in the map.
I did this so that the callback can record the results of the test, but still allow us to validate those results in the main thread, since that's where we'll want to fail an assertXXX() so that JUnit will properly fail the test.